docs/development/pigs/pig-005.rst

```diff
@@ -0,0 +1,183 @@
+committee (gammapy-coordination-l@in2p3.fr).
+
+We plan to continue the **weekly developers calls** every Friday 10 am. In addition
+we could start **monthly Gammapy user calls**, for regular user support and feedback.
+(to be discussed).
+
+We plan to hold **Gammapy workshops and tutorials** at upcoming science and
+collaboration meetings (to be disccused). 
+
+
+Use Cases
+=========
+The following use cases should be supported by Gammapy 1.0:
+
+* Fit 1D spectrum and 3D cube simultaneously
+* Further definitions defined by CTA (add link).
+
+
+Projects
+========
+The actual **development work will be structured in projects**. Each project is
+tackled by a team of (at least) two developers. They take over **responsibility
+for writing a PIG document** for the project as well as **take care of its actual
+implementation**. The PIG will be written in close **collaboration with the lead
+development team**. For the implementation we recommend a workflow where typically
+one person works on the implementation while the other is available for discussion
+and code review. We have defined the following projects:
+
+
+Maintenance and Code Quality (team effort)
+------------------------------------------
+Continue the clean up process of Gammapy. Improve code, test coverage and test
+quality in general. Change to a more uniform code style for tests. Reduce runtime
+of tests. Implement required bugfixes. Maintenance is as important as adding new
+features, but will be mostly taken over by experienced developers.
+
+Improve the Gammapy development workflow. Improve developer documentation. Define
+Github labels, projects and milestones to reflect the content of the roadmap (Axel).
+
+
+Data and Observation handling (add responsible person)
+------------------------------------------------------
+Implement support for good time intervals (GTIs). Simplify DL3 data access and
+simpify creation of custom index files.
+
+
+IRFs (add responsible person)
+-----------------------------
+Clean up and partly redesign the `gammapy.irf` sub-package. Implement IRF coordinate
+handling, unify axis handling with `gammapy.maps`. Evaluate the use of maps to store
+IRFs. Work on the IRF interface and data formats in close collaboration with
+`ctapipe`.
+
+
+Maps (add responsible person)
+-----------------------------
+Unify coordinate and unit handling in `gammapy.maps`. Migrate the healpix code
+from `healpy` to `astropy_healpix`. Finish implementation of multi-resolution
+maps (low priority).
+
+
+Map Analysis (add responsible person)
+-------------------------------------
+Unify and improve integration of background and exposure maps along the energy
+axes. Improve performance of the model evaluation by using bounding boxes and
+caching (low priority). Add support for healpix maps (low priority). Implement
+3D background model creation. Better expose classical image based background
+methods such as ring- and adaptive ring-backround.
```

Comment:
One feature that we still need to develop is spectral points from 3D analysis. Suggest to add it here in the map analysis project.

Reply:
Done